dev.sh

```diff
@@ -1,5 +1,59 @@
 #!/bin/bash
 
+REQUIRED_NODE_VERSION=22.0.0
```

Comment:
I am interested to know how you found these version's are correct and what is someone have higher version ?

Reply:
I think his check uses the required version as the minimum version.
Actually, this is a good spot because the variables should be called `MINIMUM_...` instead of `REQUIRED_...`.